external/os-autoinst-common/.github/workflows/perl-critic.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: 'Perl critic'
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+
+ perl-critic-checks:
```

Comment:
```suggestion
jobs:
  perl-critic-checks:
```

Reply:
This PR is just a subrepo update.
Any changes in external should bereviewed and fixed over there.

Reply:
I know, the problem was introduced in https://github.com/os-autoinst/os-autoinst-common/pull/32